packages/office-ui-fabric-react/src/components/ComponentExamples.test.tsx

```diff
@@ -23,9 +36,11 @@ expect.extend({
 
     if (!snapshotState) {
       // if this is a first test that want to use this snapshot, let's create it
-      snapshotState = new jestSnapshot.SnapshotState(absoluteSnapshotFile, { snapshotPath: absoluteSnapshotFile });
-      // without this, jest-snapshot will not save new snapshots.
-      snapshotState._updateSnapshot = 'new';
+      // We have to grab global state's _updateSnapshot setting to make sure jest configuration is honored
+      snapshotState = new jestSnapshot.SnapshotState(absoluteSnapshotFile, {
+        snapshotPath: absoluteSnapshotFile,
+        updateSnapshot: globalSnapshotState._updateSnapshot
```

Comment:
Will `globalSnapshotState` have a chance of being undefined here?

Reply:
There shouldn't be, but I'm failing hard. If it's ever undefined it will need to be fixed.